appengine/loopback/package.json

```diff
@@ -21,7 +21,7 @@
     "loopback-component-explorer": "5.4.0",
     "serve-favicon": "2.5.0",
     "strong-error-handler": "3.2.0",
-    "loopback-datasource-juggler": "4.5.3",
+    "^loopback-datasource-juggler": "4.5.3",
```

Comment:
is this some special syntax, or a typo? I expected `"loopback-datasource-juggler": "^4.5.3"`

Reply:
Yikes, thanks for noticing.

Reply:
How did the tests pass though? 🙈